omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/AllergyResource1_9.java

```diff
@@ -20,7 +20,7 @@
 import org.openmrs.module.webservices.rest.web.annotation.Resource;
 import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.AllergyResource1_8;
 
-@Resource(name = RestConstants.VERSION_1 + "/allergy", supportedClass = Allergy.class, supportedOpenmrsVersions = {"1.9.*", "1.10.*", "1.11.*", "1.12.*"})
+@Resource(name = RestConstants.VERSION_1 + "/allergy", order = 90, supportedClass = Allergy.class, supportedOpenmrsVersions = {"1.9.*", "1.10.*", "1.11.*", "1.12.*"})
```

Comment:
This is due to some confusion I caused.
At the time, I mistakenly combined this task with the upgrade for Hibernate 4.0.
Darius suggested to drop support for 1.8 since 1.8 did not support DbSessionFactory, and I couldn't see at the time how to support that upgrade.
When you change the resources from 1.8 to 1.9 the conflict requires the order to resolve.
I guess I can redo the change and only leave the code to support the allergyapi change.
My understanding now indicates that this should be the recommended solution. The Hibernate upgrade should be a different commit and also there are better ways to support those sorts of changes.
What do you suggest?